nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -267,4 +306,23 @@ static DataFile makeDataFile(Table icebergTable, String fileLocation) {
         .withFileSizeInBytes(Files.localInput(fileLocation).getLength())
         .build();
   }
+
+  protected static List<String> metadataVersionFiles(String tablePath) {
```

Comment:
This function (and the ones below) looks unused?

Reply:
These are used in the test code (`TestNessieView`, `TestNessieTable`)

It was moved from `TestNessieTable`. So, it can also be used for Views.